content/en/docs/tasks/run-application/configure-pdb.md

```diff
@@ -277,7 +277,8 @@ Pods in `Pending`, `Succeeded` or `Failed` phase are always considered for evict
 ## Arbitrary Controllers and Selectors
 
 You can skip this section if you only use PDBs with the built-in
-application controllers (Deployment, ReplicationController, ReplicaSet, and StatefulSet),
+application controllers (Deployment, ReplicationController, ReplicaSet, and StatefulSet)
+or controllers that implement a [scale subresource](/docs/tasks/extend-kubernetes/custom-resources/custom-resource-definitions/#scale-subresource),
 with the PDB selector matching the controller's selector.
```

Comment:
the whole sentence is becoming convoluted a bit. Should I itemize it?

Reply:
from my side it looks good